client/src/app/site/pages/meetings/pages/meeting-settings/pages/meeting-settings-group-detail/components/meeting-settings-group-detail-field/meeting-settings-group-detail-field.component.ts

```diff
@@ -162,10 +162,11 @@ export class MeetingSettingsGroupDetailFieldComponent extends BaseComponent impl
         });
         this.translatedValue = this.value ?? this.meetingSettingsDefinitionProvider.getDefaultValue(this.setting);
         if (
-            (typeof this.value === `string` && this.value !== ``) ||
-            this.setting.type === `string` ||
-            this.setting.type === `markupText` ||
-            this.setting.type === `text`
+            ((typeof this.value === `string` && this.value !== ``) ||
+                this.setting.type === `string` ||
+                this.setting.type === `markupText` ||
+                this.setting.type === `text`) &&
+            this.setting.type !== `choice`
```

Comment:
Can you move this into the first pair of brackets where the value is checked? For the other cases, this is irrelevant, since the type is asserted to be something else anyway.